perseo-main/src/test/java/com/telefonica/iot/perseo/UtilsTest.java

```diff
@@ -36,22 +36,24 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.logging.Level;
-import java.util.logging.Logger;
 import javax.servlet.ServletContext;
 import org.json.JSONObject;
 import org.junit.After;
 import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import static org.junit.Assert.*;
 
 /**
  *
  * @author brox
  */
 public class UtilsTest {
-
+	private static final Logger logger = LoggerFactory.getLogger(UtilsTest.class);
```

Comment:
Should be LOGGER? As in perseo-main/src/test/java/com/telefonica/iot/perseo/TimeRulesStoreTest.java

(The same in general for all the other similar files)

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823